### PR TITLE
Minimal config

### PR DESCRIPTION
**Pull request by matthewlehner**

### CODE REVIEW

**Comment by matthewlehner** [comment]
This is GTG, I think.

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 </head>
 <body>
-  <div id="app">Loading...</div>
+  <main id="app">Loading...</main>
```
Comment (jjt) [inline index.html]: Why `main` instead of `div`?

Comment (matthewlehner) [inline index.html]: For the CSS – changed the selector from `.app` to `main` so app styling doesn't rely on class names being present.

File: src/styles/app.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* Base Application Styles */
 
-.app {
+main {
   color: #666;
```
Comment (matthewlehner) [inline src/styles/app.css:4]: :wave:

Comment (jjt) [inline src/styles/app.css:4]: What about using `#app` instead? Would rather stick to traditonal html elements like `<div>` to not throw another curve at them.

File: src/components/app.js [hunk 3]
```diff
@@ -1,57 +1,10 @@
...
-
-  handleTodoItemClick: function (index) {
-    var todos = this.state.todos;
-    todos[index].done = !todos[index].done;
-    this.setState({todos: todos});
-  },
-
   render: function () {
     return (
       <div className="app">
```
Comment (jjt) [inline src/components/app.js:6]: Remove `className`? ;)

Comment (jjt) [inline src/components/app.js:6]: Eff it, we'll take it out in another PR.

**Pull request merged**